src/PageBanner/PageBanner.scss

```diff
@@ -0,0 +1,47 @@
+.pgn__pageBanner-component {
+  width: 100%;
+  min-height: 36px;
+  max-height: 56px;
```

Comment:
silly me, should these `px` be `em`?

Reply:
same question w/ everywhere else I used `px`

Reply:
only instances I see of max-height in paragon use vh or percentage, so not sure... @abutterworth ?

Reply:
Should this component have a `max-height` at all? What happens if the content of the `PageBanner` is multiple lines in length or otherwise has children that expands the height beyond 56px? 

IMO, this component should not enforce a max-height 😄

Reply:
@adamstankiewicz I added a max height because it was a measurement that was called out in the design, but I'd agree with you here